expr/parse.go

```diff
@@ -112,12 +112,13 @@ func Parse(e string) (*expr, string, error) {
 		return parseConst(e)
 	}
 
-	if strings.HasPrefix(e, "True") || strings.HasPrefix(e, "true") {
-		return &expr{bool: true, str: e[:4], etype: etBool}, e[4:], nil
-	}
-
-	if strings.HasPrefix(e, "False") || strings.HasPrefix(e, "false") {
-		return &expr{bool: false, str: e[:5], etype: etBool}, e[5:], nil
+	if val, wasBool := strToBool(e); wasBool {
```

Comment:
i'm confused about how we support strings supplied by the user that have quotes.
why are we not comparing `e` to `"\"False\""` anywhere for example?